airflow/models/abstractoperator.py

```diff
@@ -106,7 +106,7 @@ def dag_id(self) -> str:
     def node_id(self) -> str:
         return self.task_id
 
-    def get_template_env(self) -> "jinja2.Environment":
+    def get_template_env(self, *, force_sandboxed: bool = False) -> "jinja2.Environment":
```

Comment:
Think we should remove support for `force_sandboxed` here so we arent changing the signature of the method psrp is overriding?

Reply:
Doesn't do any harm since its an optional arg, but for the sake of a smaller diff, yes we could remove it.

Reply:
+1 to removing it since it’d make more obvious why we call the DAG’s `get_template_env` instead (so we can pass this option). It’d be nice to have a comment there describing why we don’t call the task’s `get_template_env` there.

Reply:
I pushed a commit for this (plus a small tweak for Mypy).